hacking/pr_labeler/label.py

```diff
@@ -225,6 +236,21 @@ def no_body_nag(ctx: IssueOrPrCtx) -> None:
     create_boilerplate_comment(ctx, "no_body_nag.md")
 
 
+def warn_porting_guide_change(ctx: PRLabelerCtx) -> None:
+    """
+    Complain if a user outside of the Release Management WG changes porting_guide
+    """
+    if ctx.pr.user.login in RELEASE_MANAGEMENT_MEMBERS:
+        return
+    matches: list[str] = []
+    for file in ctx.pr.get_files():
+        if file.filename.startswith("docs/docsite/rst/porting_guides/"):
```

Comment:
Files starting with `porting_guide_(core|base)` shouldn't be considered.

Also porting guides for older Ansible versions will never be updated again, so for them no warning should be printed as well.

Reply:
I don't think we should update old porting_guides either. They are historical documents that should be left alone, and we should leave the entries from collections' changelogs alone in the same way we don't patch their code.

Reply:
We did just update some of the old stuff to fix spelling errors. I can't  think of an example where we updated an EOL porting guide for anything significant, tho.

Reply:
I had to fix a broken reference in one of them recently to fix CI in this repo.

Reply:
hmm... that said... are we even able to regenerate older porting guides if something dramatic is noticed and needs a fix? Would this label just add a comment but still allow us to merge some critical PR if necessary for an EOL porting guide?

Reply:
This doesn't block anything. It simply adds a comment. We can always ignore it, but at least the warning is prominent.

Reply:
cool thanks